composer.json

```diff
@@ -27,10 +27,12 @@
         "drush/drush": "^11"
     },
     "require-dev": {
+        "behat/behat": "^3.12",
         "behat/mink-goutte-driver": "^2.0",
+        "dmore/behat-chrome-extension": "^1.4",
         "drupal/core-dev": "^10",
         "drupal/drupal-extension": "^5@alpha",
-        "palantirnet/the-build": "^4@beta"
+        "palantirnet/the-build": "dev-headless-chrome"
```

Comment:
We will want to merge the-build change first, and then update this.

Reply:
@agentrickard I tagged a new beta release of the-build: https://github.com/palantirnet/the-build/releases/tag/4.0-beta5